its/src/org/sonarlint/eclipse/its/reddeer/views/RuleDescriptionView.java

```diff
@@ -50,7 +50,7 @@ public Label getRuleSeverity() {
   }
 
   public Label getRuleKey() {
-    return new DefaultLabel(getCTabItem(), 8);
+    return new DefaultLabel(getCTabItem(), 5);
```

Comment:
As we now have the same number of direct child elements in the `RuleDescriptionView` this change done earlier is reverted